client/Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
```

Comment:
don't think you need a Pipfile in `/client` folder. may just have been created by accident!

Reply:
Yes, maybe created by accidentally. Because I did not create it.